uberfire-api/src/main/java/org/uberfire/workbench/type/ResourceTypeDefinition.java

```diff
@@ -68,4 +70,10 @@
      * @return true if matches, otherwise false
      */
     boolean accept(final Path path);
+
+    /**
+     * Defines the category for each resource. If no category is defined a default is set
```

Comment:
Please add a comment about what category would be the default one if not defined..